frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts

```diff
@@ -526,6 +630,71 @@ describe('Serving Runtime List', () => {
       kserveModal.findSubmitButton().click();
       kserveModal.shouldBeOpen(false);
 
+      // dry run request
+      cy.wait('@createServingRuntime').then((interception) => {
+        expect(interception.request.url).to.include('?dryRun=All');
+        expect(interception.request.body).to.eql({
+          apiVersion: 'serving.kserve.io/v1alpha1',
+          kind: 'ServingRuntime',
+          metadata: {
```

Comment:
@christianvogt  Is there any better way to assert the request body? This file has a lot of different request body to assert. I'm unsure how to handle this. Is it okay to assert it like this or using the mock by modifying would be the better option here?

Reply:
Feel free to assert specific values in the spec instead of the entire spec itself. Those that pertain specifically to the inputted values would suffice.